### PR TITLE
[Merged by Bors] - set cache to 100_000 and disable libp2p resource manager

### PR DESCRIPTION
**Pull request by dshulyak**
- raising atx cache size to 100_000 to prevent threshing of the lru cache
- disable libp2p resource manager. we are not relying on it

### CODE REVIEW

File: datastore/store.go [hunk 1]
```diff
@@ -237,7 +237,10 @@ func (db *CachedDB) GetEpochWeight(epoch types.EpochID) (uint64, []types.ATXID,
 }
 
 // IterateEpochATXHeaders iterates over ActivationTxs that target an epoch.
-func (db *CachedDB) IterateEpochATXHeaders(epoch types.EpochID, iter func(*types.ActivationTxHeader) error) error {
+func (db *CachedDB) IterateEpochATXHeaders(
+	epoch types.EpochID,
+	iter func(*types.ActivationTxHeader) error,
+) error {
```
Comment (fasmat) [inline datastore/store.go:243]: Did you change your formatting settings? I don't mind lines that are 120 (or even 160) characters long. Limiting ourselves to 80 characters seems archaic to me.

**Review by fasmat — APPROVED** [review]

File: datastore/store.go (hunk 1 above)
Comment (dshulyak) [reply datastore/store.go:243]: i installed golines per @poszu request :) 

i think default is 100, i overlooked that. @poszu do you prefer 200 or 160?

Comment (dshulyak) [reply datastore/store.go:243]: will merge it as is (because i cherrypicked it to v1.2.2) and update config for next patches

**Comment by dshulyak** [comment]
bors merge

File: datastore/store.go (hunk 1 above)
Comment (poszu) [reply datastore/store.go:243]: @dshulyak I prefer up to 120. Longer lines don't fit in side-by-side reviews. 80 is in fact too short.

**Pull request closed without merging**